src/components/common/button/button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
No need for the `common` folder. The `components` folder should be enough for this